figures/scripts/figS2-snv-callers-panels.R

```diff
@@ -135,30 +131,27 @@ for (dataset in c("tcga", "pbta")) {
 
   ## Upset plots -------------------------------------------------------
   print("Preparing data for upset plot")
-  detect_mat <- all_caller_df %>%
-    # Bring over VAF columns and the index
+  vaf_mat <- all_caller_df %>%
+    # Bring over VAF columns
     select(starts_with("VAF_")) %>%
     as.matrix()
 
-  # Store the indices as dimnames
-  dimnames(detect_mat)[[1]] <- all_caller_df$index
+  # Turn into logical matrix
```

Comment:
A logical matrix of what? I actually liked the previous name, because the VAF data is gone at this point.
```suggestion
  # Turn into logical matrix of detected or not (NA means the mutation was not detected)
```